purchase_analytic_distribution/__openerp__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+#  Copyright (C) 2004-2010 Tiny SPRL (<http://tiny.be>).
+#  Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Purchase Analytic Plans",
+    "version": "9.0.1.0.0",
+    "category": "Purchase Management",
+    "author": "Odoo SA, "
```

Comment:
Are you sure Odoo puts their authorship this way? Check default value of author key in code, which is the one they use.